service/static/index.html

```diff
@@ -11,6 +11,8 @@
     <div class="container">
       <div class="page-header">
           <h1>Product Demo REST API Service</h1>
+          <!-- <p>API Documentation is <a href='/apidocs'>here</a></p> -->
+          <p>API Documentation is <a href='/apidocs'>here</a></p>
```

Comment:
@mpourostad 
Should we set the target as well ? So that whenever someone clicks on it, A new Tab gets opened.
Example :`<p>API Documentation is <a href='/apidocs' target="_blank">here</a></p>`

Reply:
Good idea! just added this in new commit